badge_server/main.py

```diff
@@ -112,6 +112,17 @@ def get_highest_status(cls, statuses: Iterable[enum.Enum]):
     BadgeStatus.SUCCESS: 'brightgreen',
 }
 
+BADGE_STATUS_TO_TEXT = {
```

Comment:
Can't these be the direct values of `BadgeStatus`?

Reply:
They could be. It seems like this is a more clean separation though.

Reply:
Hmm... it just seemed a bit redundant since the enum values for BadgeStatus are constant strings anyway. I think having the `_badge_status_to_text()` function would be sufficient for clarity?

Reply:
OK, fixed.